helper.sh

```diff
@@ -289,14 +289,18 @@ function get_result_dir() {
     declare -a batch_sizes_arr=("${!2}")
     declare -a distribution_types_arr=("${!3}")
     local device_type_arg=$4
+    local norm=""
+    if [[ ${is_normalized} == true ]]; then
+        norm="-normalized"
+    fi
 
 
     local result_base=$(IFS=- ; echo "${models_arr[*]}")
     for i in "${!models_arr[@]}"; do
         concatenated_string="${concatenated_string}${models_arr[$i]}-${batch_sizes_arr[$i]}-${distribution_types_arr[$i]}_"
     done
     local result_id=${concatenated_string%_}
-    result_dir=results/${device_type_arg}/${result_base}/${result_id}
+    result_dir=results/${device_type_arg}/${result_base}${norm}/${result_id}
```

Comment:
@psinha25, if you want to change the `${device_type_arg}` to `model-formulation-datapoints`, please do so, but be careful about the `${norm}` added. It is needed.